src/test/java/org/broadinstitute/hellbender/utils/tsv/SimpleCSVWriterWrapperWithHeaderUnitTest.java

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.hellbender.utils.tsv;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public class SimpleCSVWriterWrapperWithHeaderUnitTest extends GATKBaseTest {
+
+
```

Comment:
Separate the cloud and local file test cases. Also add tests covering `setRow()`, `fill()`, the version of `setColumn()` that takes an index, and the other methods you're not currently testing.